Tutorials_UsingAPIs/Flow-Control-EventQueue/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"
+#include "mbed_events.h"
```

Comment:
This is already included in "mbed.h".
If you insist on re-including here, it is a good idea to include with the directory to give some kind of namespace (i.e `#include "events/mbed_events.h"`).

Reply:
I'll change it. It was copied from another snippet.